src/bartholomew.rs

```diff
@@ -141,13 +141,26 @@ pub fn render(req: Request) -> Result<Response> {
                         .clone()
                         .unwrap_or_else(|| DEFAULT_CONTENT_TYPE.to_owned());
 
+                    let cache_control = doc
+                        .head
+                        .cache_control
+                        .clone()
+                        .unwrap_or_else(|| DEFAULT_CACHE_CONTROL.to_owned());
```

Comment:
This is a change in behaviour, I think, defaulting to instructing `no-cache` instead of not caring?  Is that intentional?  If so, maybe leave a comment on why?

Should users have a way to omit the `cache-control` header instead of using `no-cache`?

Reply:
Good point. I will just make the adding of the header optional.